app/views/bars/edit.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1>Edit <%= @bar.name %></h1>
+
+<p><%= @bar.address %></p>
```

Comment:
These should probably be `<div>` tags when you get into styling this instead of `<p>` tags.